app/src/main/java/com/jerboa/ui/components/common/DrawerItems.kt

```diff
@@ -73,6 +74,61 @@ fun IconAndTextDrawerItem(
     }
 }
 
+@Composable
+fun IconAndTextAndCheckboxDrawerItem(
+    text: String,
+    icon: ImageVector? = null,
+    iconBadgeCount: Int? = null,
+    highlight: Boolean = false,
+    checked: Boolean,
+    onChecked: (Boolean) -> Unit,
+) {
+    val spacingMod = Modifier
```

Comment:
Since this is just a single modifier I'm not sure it needs to be a value like this.